### ISSUE TITLE
OpenTelemetry extension classes

### ISSUE THREAD

**Issue by xvilo** [shipmonk-rnd/composer-dependency-analyser#143]
How to deal with classes/FQCN and such that are provided from extensions? We use the OTEL PHP Extension and get the following error:
```
Unknown functions!
(those are not declared, so we cannot check them)
  • OpenTelemetry\Instrumentation\hook
    in src/Instrumentation/Frontend/FrontendInstrumentation.php:53
```

**Comment by janedbal** [comment]
Anything that is not autoloadable (classes) or visible in `get_defined_functions` is reported as unknown. I'm not sure how this extension approaches visibility of its symbols, but we are dealing with similar issue wich DataDog symbols (which also provide telemetry), so we just ignored those:

```php
return $config
    ->ignoreUnknownFunctionsRegex('~^DDTrace~')
    ->ignoreUnknownClassesRegex('~^DDTrace~')
```

Normal extensions (like ext-dom) have their symbols loaded and available since PHP startup. 

If you are able to initialize those symbols somehow manually, you can do that in the PHP config. We are doing similar thing [here](https://github.com/shipmonk-rnd/phpstan-rules/blob/master/composer-dependency-analyser.php#L6) (this example prepends new autoloader to composer's).

**Comment by janedbal** [comment]
Actually... this might be a bug on our side, let me check. I'll gid deeper into the issue with DD.

**Comment by janedbal** [comment]
Turns out, extension functions were not properly excluded (case sensitivity mismatch), fixed in #144. Classes just need to be autoloadable.

Can you try dev-master if that fixes your problem? Thank you
